meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/ccsp-common-library.bbappend

```diff
@@ -20,16 +20,29 @@ do_install_append_class-target() {
    fi
    install -D -m 0644 ${S}/systemd_units/wan-initialized.target ${D}${systemd_unitdir}/system/wan-initialized.target
    install -D -m 0644 ${S}/systemd_units/wan-initialized.path ${D}${systemd_unitdir}/system/wan-initialized.path
+    #Telemetry support
```

Comment:
are you able to see results in cloud .https://jira.rdkcentral.com/jira/projects/RDKCLOUD/issues/RDKCLOUD-30?filter=allopenissues

Reply:
We confirmed ,this functionality is working based on logs .
 As we know, log upload portal is not working for few weeks.
Also, required logs are already uploaded in JIRA .